packages/test-utils/transforms/cssTransform.cjs

```diff
@@ -7,7 +7,9 @@
 
 module.exports = {
   process() {
-    return 'module.exports = {};';
+    return {
+      code: 'module.exports = {};'
+    };
```

Comment:
here and in fileTransform.cjs, this change is required by the upgrade to jest 28: https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28#transformer